### PR TITLE
Fix vlan incremental config and add vs test cases

### PR DESCRIPTION
**Pull request by jipanyang**
Signed-off-by: Jipan Yang <jipan.yang@alibaba-inc.com>

**What I did**
Fix vlan incremental config update.

**Why I did it**

**How I verified it**
VS.
**Details if related**

### CODE REVIEW

**Comment by lguohan** [comment]
can you add more comments on what problem it is trying to solve here?

File: cfgmgr/vlanmgr.cpp [hunk 1]
```diff
@@ -251,11 +251,14 @@ void VlanMgr::doVlanTask(Consumer &consumer)
             string members;
 
             /*
-             * Don't program vlan again if state is already set.
-             * will hit this for docker warm restart.
-             * Just set the internal data structure and remove the request.
+             * If state is already set for this vlan, but it doesn't exit in m_vlans set,
```
Comment (qiluo-msft) [inline cfgmgr/vlanmgr.cpp]: >exit [](start = 69, length = 4)

typo?

**Comment by jipanyang** [comment]
"vconfig" missing in the Jenkins unit test environment,  it is requirement for the tagged vlan test.

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
vconfig is installed.

**Comment by jipanyang** [comment]
@qiluo-msft  as discussed offline, please help merge the change if no further comment. thanks.

**Comment by qiluo-msft** [comment]
Resolve conflict?

**Comment by jipanyang** [comment]
@qiluo-msft conflict resolved.

**Comment by qiluo-msft** [comment]
@lguohan Do you have more concern?

**Pull request merged**